app-blank-after-kendo-upgrade.md

```diff
@@ -0,0 +1,43 @@
+
+---
+title: Upgrading from Kendo UI Core to Professional Breaks App
+description: After upgrading an app through the Package Manager to use Kendo UI Professional it appears blank on start up. 
+type: troubleshooting
+page_title: Upgrading from Kendo UI Core to Professional Breaks App
+slug: app-broken-after-kendo-upgrade
+position: 0
+tags: kendo, package manager, blank, references, undefined
+ticketid: 1017633
+
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Package Manager</td>
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>
+  </tr>
+  <tr>
+    <td>Code (AppBuilder) client</td>
+    <td>All</td>
+  </tr>
+</table>
+## Description
+When the Kendo UI package type is changed from Kendo UI Core to Kendo UI Professional (and vice versa) in an app using the Package Manager, all styles and functionality may be lost.
```

Comment:
Two things need to be noted here:
* Vice versa is usually to be avoided in technical documentation because this falls in the foreign/complex/ambiguous category. 
* Passive voice is to be avoided. A few cases when you might want to use passive voice explicitly is when you want to emphasize on the action (rather than the person who performs it) or when the who of the action is unknown.

Consider rewriting this sentence into something like this:

When you switch between Kendo UI Core and Kendo UI Professional in your app using the Package Manager, you may lose all style and functionality.